### PR TITLE
Add ceil/floor operations.

### PR DESCRIPTION
**Pull request by dan-zheng**
Add ceil/floor and their adjoints.

The adjoints of ceil/floor are 0:
* TensorFlow gradients: [ceil](https://github.com/tensorflow/tensorflow/blob/a89726dea8d9005a5f9ca73ad14f28c32cd87e56/tensorflow/python/ops/math_grad.py#L1151), [floor](https://github.com/tensorflow/tensorflow/blob/a89726dea8d9005a5f9ca73ad14f28c32cd87e56/tensorflow/python/ops/math_grad.py#L1146)
* PyTorch derivatives: [ceil](https://github.com/pytorch/pytorch/blob/43b151224e5487eea00c7ce4e0efd442ee623715/tools/autograd/derivatives.yaml#L169), [floor](https://github.com/pytorch/pytorch/blob/43b151224e5487eea00c7ce4e0efd442ee623715/tools/autograd/derivatives.yaml#L270)

### CODE REVIEW

**Review by rxwei — APPROVED** [review]
LGTM. Thanks!
